service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/twin/TwinCollection.java

```diff
@@ -329,8 +330,13 @@ private static void addMetadata(TwinCollection twinCollection, Map<? extends Str
      *
      * @return The {@code JsonElement} with the content of this class.
      */
-    public JsonElement toJsonElement() {
-        return ParserUtility.mapToJsonElement(this);
+    public JsonObject toJsonObject() {
+        JsonObject object = ParserUtility.mapToJsonElement(this).getAsJsonObject();
+        if (this.version != null)
+        {
+            object.add(VERSION_TAG, new JsonPrimitive(this.version));
```

Comment:
How about changing to 
```suggestion
            object.addProperty(VERSION_TAG, this.version);
```
?

Reply:
I like that better, yeah. Good idea